source/installation_guide/upgrading/index.rst

```diff
@@ -15,6 +15,7 @@ And that may be a good thing, since it can expose problems which could otherwise
    :maxdepth: 1
 
    from-2.3-to-2.4
+   from-2.3-to-2.3.12
```

Comment:
How will this work? Is there going to be a from-2.3-to-2.3.X for each sufficiently major release? Or are we going to make the next one from-2.3.12-to-2.3.X? IIRC there was some pushback in the past about having too many of these.

Reply:
Previously we have just described these kind of changes in the NEWS file. Although it has been rare that we have broken config syntax in the middle of v2.x releases..

But yeah, I'd rather see a single page here like from-2.3-to-2.3.x which lists all the things changed since v2.3.0. Otherwise if you skip multiple versions you'll need to click a ton of times to see all the changes.